server/server.go

```diff
@@ -112,3 +113,73 @@ func (s *Server) SumCalories(id string) (int, error) {
 	}
 	return sum, nil
 }
+
+// CheckDB : return a boolean check on the database structure
+func (s *Server) CheckDB() bool {
+	sqlStatement := `
+	SELECT column_name, data_type
+	FROM information_schema.columns
+	WHERE table_name = 'users'`
+	rows, err := s.db.Query(sqlStatement)
+	if err != nil {
+		log.Print("CheckDB on Users failed: " + err.Error())
+		return false
+	}
+
+	var col string
+	var typ string
+
+	UsersSchema := []string{"user_id", "max_cal", "timezone", "name"}
+	UsersSchemaType := []string{"bigint", "bigint", "integer", "text"}
+
+	defer rows.Close()
+
+	idx := 0
+	for rows.Next() {
```

Comment:
Sorry wasn't too clear - `defer` should be closest, since it is immediately relevant, so something like:
```go
idx := 0
defer rows.Close()
for rows.Next() { ... }
```

same for the other one